go/mysql/auth_server_static.go

```diff
@@ -36,7 +37,7 @@ import (
 var (
 	mysqlAuthServerStaticFile   = flag.String("mysql_auth_server_static_file", "", "JSON File to read the users/passwords from.")
 	mysqlAuthServerStaticString = flag.String("mysql_auth_server_static_string", "", "JSON representation of the users/passwords config.")
-	mysqlAuthRotationTime       = flag.Duration("mysql_auth_rotation_time", 300*time.Second, "Cooldown period before reloading json config.")
+	mysqlAuthRotationTime       = flag.Int64("mysql_auth_rotation_time", 300, "Cooldown period before reloading json config.")
```

Comment:
I think all time interval flags should be specified as durations like you had it before.
You can always convert to seconds in the code.